lib/msal-common/src/telemetry/performance/PerformanceEvent.ts

```diff
@@ -221,6 +221,12 @@ export type StaticFields = {
     matsHttpEventCount?: number;
     httpVerToken?: string;
     httpVerAuthority?: string;
+
+    // Broker timeouts
```

Comment:
Can you add a bit more context here? i.e. "This is used by the 1P browser lib". I do wonder though if there's a way we can avoid exposing these types of details to the 3P/common libs at all?

Reply:
Not at the moment unfortunately, but we should be able in the future by extending 3p telemetry classes in 1p. Added context.